src/api/proto/cloudpb/cloudapi.proto

```diff
@@ -956,6 +956,8 @@ message ConfigForVizierRequest {
   vizierconfigpb.VizierSpec vz_spec = 2;
   // Kubernetes version of the cluster Vizier is running on.
   string k8s_version = 3 [ (gogoproto.customname) = "K8sVersion" ];
+  // ID of the Vizier (Cluster ID).
+  string vizier_id = 4 [ (gogoproto.customname) = "VizierID" ];
```

Comment:
I'm not knowledgable on cloud so my comment should be vetted, but I think we will want to update [this](https://github.com/pixie-io/pixie/blob/26a31df536614d5e75133dc3f39e5c8573382834/src/cloud/api/controllers/config_grpc.go#L43) ConfigForVizierRequest from cloud to the config manager service in this review.

Without that change, I believe the next step of our plan (to modify the operator to provide a vizier id on update) won't be able to have its VizierID parameter propagated fully through to the config manager service.

cc @vihangm

Reply:
That is correct, we need to add VizierID to this proto as well: https://github.com/pixie-io/pixie/blob/26a31df536614d5e75133dc3f39e5c8573382834/src/cloud/config_manager/configmanagerpb/service.proto#L38

Reply:
My bad, I was just going through the order for comments. I see this was done already in the PR!

Reply:
Sorry I think I actually misunderstood your comment for a sec (thought you were talking about the protos). We do need to update that file you referenced above to pass the vizierID along in the ConfigForVizierRequest.